src/skmultiflow/rules/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .vfdr import VFDR
```

Comment:
Add a description for this module:

```
"""
The :mod:`skmultiflow.rules` module contains ...
"""
```

This is used for generating the documentation

Reply:
Created #112 to address this